src/editors/editDocumentCodeLensProvider.ts

```diff
@@ -38,17 +40,20 @@ implements vscode.CodeLensProvider {
     // Show code lenses only for the list of documents or a single document
     // that are returned by the find() method.
     if (type === 'Cursor') {
-      this._updateCodeLensesForCursor(data);
+      codeLensesInfo = this._updateCodeLensesForCursor(data);
     } else if (type === 'Document') {
-      this._updateCodeLensesForDocument(data);
+      codeLensesInfo = this._updateCodeLensesForDocument(data);
     }
+
+    this._codeLensesInfo = codeLensesInfo;
+    this._onDidChangeCodeLenses.fire();
```

Comment:
Not sure I follow - what does this firing do here? does it update it on subsequent runs?

Reply:
It can be removed 👍 Leftover from refactoring.